### DISCUSSION TITLE
UTM SE on the iOS App Store

### DISCUSSION THREAD

**Discussion by upintheairsheep** [utmapp/UTM#6257] [Ideas]
Hello, Apple finally is allowing game emulators on the iOS App Store that download emulated games (not only in EU or EEA) after legal pressure (possibly driven by the DoJ antitrust suit regarding “super apps” like X (formally known as Twitter) or the EU). UTM SE should be able to be reviewed again, maybe packing in small games like bootchess.

**Comment by lgrachov** [comment]
It is officially published on the App Store! Here it is. https://apps.apple.com/ua/app/utm-se-retro-pc-emulator/id1564628856

**Reply by aehlke** [reply]
Is this open sourced as well?

**Reply by lgrachov** [reply]
> Is this open sourced as well?

Yes. It is just a bit minified (some things removed to get it approved). You can just remove a few things and it will be the same.
